plugins/dfu/fu-dfu-target-stm.c

```diff
@@ -6,6 +6,8 @@
 
 #include "config.h"
 
+#include <fwupd.h>
+#include <fwupdplugin.h>
```

Comment:
you don't need the former if you have the latter -- there's a few of these here too

Reply:
I've added a precommit script to detect this; feel free to roll it in with yours. Thanks for all the docs work; it's looking great.